doc/install/build-ceph.rst

```diff
@@ -61,15 +61,14 @@ Advanced Package Tool (APT)
 
 To create ``.deb`` packages for Debian/Ubuntu, ensure that you have cloned the 
 `Ceph`_ repository, installed the `Build Prerequisites`_ and installed 
-``debhelper``::
+``debhelper`` and ``dpkg``::
 
-	sudo apt-get install debhelper
+	sudo apt-get install debhelper dpkg
```

Comment:
dpkg is an essential package in debian derivative distros. i don't think we need to install specifically.

and debhelper is listed by `Build-Depends` in d/control, so probably "`Build Prerequisites`" is good enough? as `install-deps.sh` installs devscripts, equivs and dpkg-dev. they will be enough to bootstrap the next step to install `Build-Depends` listed by debian/control.